.env.example

```diff
@@ -1 +1,3 @@
-PORT=3000
```

Comment:
why is this variable removed ? please don't.

Reply:
Oh sorry , I will fix that

Reply:
solved that problem, please review.